### PR TITLE
Removed smoke on RHEL 7.4 and added on ubuntu 18.04

### PR DESCRIPTION
**Pull request by yuriw**
Signed-off-by: Yuri Weinstein <yweinste@redhat.com>

- [ ] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

### CODE REVIEW

File: qa/crontab/teuthology-cronjobs [hunk 1]
```diff
@@ -31,7 +31,7 @@ CEPH_QA_EMAIL="ceph-qa@lists.ceph.com"
...
 0 5  * * * CEPH_BRANCH=master; MACHINE_NAME=ovh; /home/teuthology/bin/cron_wrapper teuthology-suite -v -c $CEPH_BRANCH -n 7 -m $MACHINE_NAME -s smoke -k testing -p 71 -e $CEPH_QA_EMAIL --distro ubuntu --distro-version 16.04
-0 7  * * * CEPH_BRANCH=master; MACHINE_NAME=ovh; /home/teuthology/bin/cron_wrapper teuthology-suite -v -c $CEPH_BRANCH -n 7 -m $MACHINE_NAME -s smoke -k testing -p 70 -e $CEPH_QA_EMAIL --distro rhel --distro-version 7.5 ~/rhel_only_on_ovh.yaml
+0 7  * * * CEPH_BRANCH=master; MACHINE_NAME=ovh; /home/teuthology/bin/cron_wrapper teuthology-suite -v -c $CEPH_BRANCH -n 7 -m $MACHINE_NAME -s smoke -k testing -p 70 -e $CEPH_QA_EMAIL --distro ubuntu --distro-version 18.04
 # run one time per week on rhel7.4
 @weekly    CEPH_BRANCH=master; MACHINE_NAME=ovh; /home/teuthology/bin/cron_wrapper teuthology-suite -v -c $CEPH_BRANCH -n 7 -m $MACHINE_NAME -s smoke -k testing -p 70 -e $CEPH_QA_EMAIL --distro rhel --distro-version 7.4 ~/rhel_only_on_ovh.yaml
 2 7  * * * CEPH_BRANCH=master; MACHINE_NAME=ovh; /home/teuthology/bin/cron_wrapper teuthology-suite -v -c $CEPH_BRANCH -n 7 -m $MACHINE_NAME -s smoke -k testing -p 70 -e $CEPH_QA_EMAIL --distro centos --distro-version 7.4
```
[the remark this answers is not in the document]
Comment (yuriw) [reply qa/crontab/teuthology-cronjobs]: @djgalloway yes and we say - remove 7.4

[the remark this answers is not in the document]
Comment (djgalloway) [reply qa/crontab/teuthology-cronjobs]: I mean line 37.  `--distro centos --distro-version 7.4`

[the remark this answers is not in the document]
Comment (yuriw) [reply qa/crontab/teuthology-cronjobs]: @djgalloway you are right I fixed, rhel, but centos still points to 7.4.  Do you suggest to change it to newer version?

[the remark this answers is not in the document]
Comment (djgalloway) [reply qa/crontab/teuthology-cronjobs]: I don't know?  That's not really up to me.  I can verify crontab syntax but I don't know what we should be testing on.

[the remark this answers is not in the document]
Comment (vasukulkarni) [reply qa/crontab/teuthology-cronjobs]: 7.4 can be removed and 7.5 used instead but in the diff i see 7.5 cronjob convert to ubuntu?

[the remark this answers is not in the document]
Comment (yuriw) [reply qa/crontab/teuthology-cronjobs]: @vasukulkarni I replaced `RHEL 7.4` with `ubuntu 18.04`

**Pull request closed without merging**